Formula/apachetop.rb

```diff
@@ -1,14 +1,10 @@
 class Apachetop < Formula
   desc "Top-like display of Apache log"
-  homepage "https://web.archive.org/web/20170809160553/freecode.com/projects/apachetop"
-  url "https://deb.debian.org/debian/pool/main/a/apachetop/apachetop_0.19.7.orig.tar.gz"
-  sha256 "88abf58ee5d7882e4cc3fa2462865ebbf0e8f872fdcec5186abe16e7bff3d4a5"
+  homepage "https://github.com/tessus/apachetop"
+  url "https://github.com/tessus/apachetop/releases/download/0.23.2/apachetop-0.23.2.tar.gz"
```

Comment:
This is same as Debian does. With using github as source, we can also default to the default github livecheck strategy